### PR TITLE
Adds configuration for vikings cluster

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

File: installation/vikings.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+							instance.TypeR38XLarge,
+						),
+						Available:    instance.Available(),
+						Capabilities: instance.Capabilities(),
+						Default:      instance.Default,
+					},
+				},
+				Route53: route53.Route53{
+					HostedZones: hostedzones.HostedZones{
+						API:     "Z1Z5J7V0K6UO20",
```
Comment (JosephSalisbury) [inline installation/vikings.go]: These three hosted zone ids will need to be updated.

**Comment by JosephSalisbury** [comment]
@rossf7 @calvix ptal

File: installation/vikings.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+		Guest: guest.Guest{
+			Hyperkube: hyperkube.Hyperkube{
+				Version: hyperkube.Version,
+			},
+			Kubectl: kubectl.Kubectl{
+				Version: kubectl.Version,
+			},
+			Kubernetes: kubernetes.Kubernetes{
+				API: kubernetes.API{
+					EndpointBase: "g8s.eu-west-1.vikings.aws.giantswarm.io",
```
Comment (rossf7) [inline installation/vikings.go]: This needs to be `k8s` to match hosted zones.

Comment (JosephSalisbury) [inline installation/vikings.go]: Fixed

**Pull request merged**